src/forms.css

```diff
@@ -444,14 +444,14 @@ because it seems to be more cross-browser consistent with all HTML5 input types
   height: 20px;
   border-radius: 50%;
   background: var(--white);
-  border: 2px solid var(--default-interactive-color);
+  border: 2px solid var(--default-secondary-interactive-color);
   cursor: pointer;
 }
 
 /* range thumb:hover */
-.range:hover::-webkit-slider-thumb { border-color: var(--default-interactive-color-dark); }
-.range:hover::-ms-thumb { border-color: var(--default-interactive-color-dark); }
-.range:hover::-moz-range-thumb { border-color: var(--default-interactive-color-dark); }
+.range:hover::-webkit-slider-thumb { border-color: var(--gray); }
+.range:hover::-ms-thumb { border-color: var(--gray); }
+.range:hover::-moz-range-thumb { border-color: var(--gray); }
```

Comment:
Lets merge this and feel it out in some real projects – we can come back to it later.